services/app-api/handlers/reports/update.ts

```diff
@@ -57,7 +57,10 @@ export const updateReport = handler(async (event, context) => {
           state,
           id,
           lastAltered: Date.now(),
-          fieldData: validatedFieldData,
+          fieldData: {
+            ...currentReport.fieldData,
+            ...validatedFieldData,
+          },
```

Comment:
Wouldn't this just write over the fieldData object with only what's being updated?

Reply:
spreading currentReport.fieldData takes care of that!